vnext/Microsoft.ReactNative.Cxx/JSI/JsiValue.h

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
>// [](start = 0, length = 2)

(nit) Should this file be named "JsiValueAbi.h" to specify that this is ABI safe?

Reply:
I have removed it for now. It will be part of another PR.

---
In reply to: [457588163](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r457588163) [](ancestors = 457588163)